tests/test_parameters.py

```diff
@@ -4,23 +4,13 @@
 import pytest
 
 from sklearn.base import clone
-from tensorflow.python.keras.testing_utils import get_test_data
+from sklearn.datasets import make_classification
 
 from scikeras.wrappers import KerasClassifier, KerasRegressor
 
 from .mlp_models import dynamic_classifier, dynamic_regressor
 
 
-# Defaults
-INPUT_DIM = 5
-HIDDEN_DIM = 5
-TRAIN_SAMPLES = 10
-TEST_SAMPLES = 5
-NUM_CLASSES = 2
-BATCH_SIZE = 5
-EPOCHS = 1
-
-
```

Comment:
`get_tet_data` was returning a multiclass-multioutput target, which is now unsupported in KerasClassifier. None of these tests require that as their target, so I replaced the data source with `sklearn.datasets.make_classification`